cmd/chainsimulator/main.go

```diff
@@ -152,13 +159,15 @@ func startChainSimulator(ctx *cli.Context) error {
 	}
 
 	numValidatorsMetaShard := ctx.GlobalInt(numValidatorsMeta.Name)
-	if numValidatorsMetaShard < 1 {
-		return errors.New("invalid value for the number of validators for metachain")
-	}
 	numWaitingValidatorsMetaShard := ctx.GlobalInt(numWaitingValidatorsMeta.Name)
 	if numWaitingValidatorsMetaShard < 0 {
 		return errors.New("invalid value for the number of waiting validators for metachain")
 	}
+	if isSovereign {
```

Comment:
Move this in a func above when you are reading the configs and then have two separate config check functions based on this bool flag

Reply:
Only one check was problematic, but added `&& !isSovereign` and problem is resolved